leaves/cursor.ml

```diff
@@ -0,0 +1,52 @@
+type t = {
+    style : Spices.style;
+    (* whether cursor is visible *)
+    focus : bool;
+
+    (* blinking parameters *)
+    blink : bool;
+    show : bool;
+    fps : Fps.t;
+  }
+
+
+open struct
+  (* Colors *)
+  let white = Spices.color "#FFFFFF"
+  let black = Spices.color "#000000"
+end
+
+let make
+      ?(style=Spices.(default |> bg white |> fg black))
+      ?(blink=true)
+      ?(fps=Fps.of_float 2.5) ()
+  = {
+    focus = true;
+    blink = blink;
+    fps = fps;
+    show = true || (not blink);
```

Comment:
should this be `not blink`?

Reply:
This should just be `true`, thanks for catching this!